lib/vm.js

```diff
@@ -26,6 +26,7 @@ const {
   kParsingContext,
   makeContext,
   isContext: _isContext,
+  getWrappedFunction
```

Comment:
nit: s/`getWrappedFunction`/`compileFunction`/

Reply:
umm, do you mean I should rename the C++ binding and well? The C++ binding is named `getWrappedFunction`//`GetWrappedFunction` while the public API function is called `vm.compileFunction`.

Reply:
Yes, that’s what I mean :)

Reply:
Sure! Will do.